src/Microsoft.AspNetCore.SignalR/HubEndPoint.cs

```diff
@@ -29,9 +29,7 @@ public class HubEndPoint<THub> : HubEndPoint<THub, IClientProxy> where THub : Hu
 
     public class HubEndPoint<THub, TClient> : EndPoint, IInvocationBinder where THub : Hub<TClient>
     {
-        private readonly Dictionary<string, Func<Connection, InvocationDescriptor, Task<InvocationResultDescriptor>>> _callbacks
-            = new Dictionary<string, Func<Connection, InvocationDescriptor, Task<InvocationResultDescriptor>>>(StringComparer.OrdinalIgnoreCase);
-        private readonly Dictionary<string, Type[]> _paramTypes = new Dictionary<string, Type[]>();
+        private readonly Dictionary<string, HubMethodDescriptor> _methods = new Dictionary<string, HubMethodDescriptor>(StringComparer.OrdinalIgnoreCase);
```

Comment:
I am weighing what @BrennanConroy said with "having two methods with different casing is not a good practice", not having to care about casing makes it easier to use, VB.NET is case insensitive (LOL). I think case insensitive is ultimately better - don't remember anyone complaining about this with SignalR 2.